packages/web/src/pages/upload-page/fields/availability/HiddenAvailabilityFields.tsx

```diff
@@ -80,11 +82,11 @@ const AvailabilityToggleField = (props: AvailabilityToggleFieldProps) => {
     type: 'checkbox'
   })
   return (
-    <div className={styles.switchRow}>
+    <label className={styles.switchRow}>
```

Comment:
should the label be on the text?

Reply:
So if you wrap the input in a label, the label becomes the text next to the input, and then you also don't need to assign the label to the input, it becomes implicit!